compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUVirtualVectorLowering.cpp

```diff
@@ -55,6 +56,10 @@ void LLVMCPUVirtualVectorLoweringPass::runOnOperation() {
           .setVectorMultiReductionLowering(vectorMultiReductionLowering)
           .setVectorTransferSplit(vectorTransferSplit);
 
+  auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(funcOp);
+  DictionaryAttr targetConfig =
+      targetAttr ? targetAttr.getConfiguration() : nullptr;
```

Comment:
IREE prefers braces for single statement, which is slightly different from LLVM style. btw, I think we can write it like below.

```suggestion
  DictionaryAttr targetConfig;
  if (auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(funcOp)) {
    targetConfig = targetAttr.getConfiguration();
  }
```

Reply:
Fixed.